### PR TITLE
[PrivacyPrivilegeManager] Add new API for checking & requesting multiple privileges

### PR DESCRIPTION
**Pull request by xerrni**
### Description of Change ###

C# bindings for checking & requesting multiple privileges using single API call

### API Changes ###
 - TCSACR-184
(It's related Native ACR-1253.)

#### Class PrivacyPrivilegeManager: ####
Method:
```csharp
public static IEnumerable<CheckResult> CheckPermissions(IEnumerable<string> privileges);
public static Task<RequestMultipleResponseEventArgs> RequestPermissions(IEnumerable<string> privileges);
```
#### Class PermissionRequestResponse: ####
Event:
```csharp
public RequestResult Result { get; internal set; }
public string Privilege { get; internal set; }
```
#### Class RequestResponseEventArgs : EventArgs ####
Property:
```csharp
public PermissionRequestResponse Response { get; }
```
#### Class RequestMultipleResponseEventArgs : EventArgs ####
Property:
```csharp
public CallCause Cause { get; internal set; }
public IEnumerable<PermissionRequestResponse> Responses { get; internal set; }
```

### CODE REVIEW

**Comment by xerrni** [comment]
@myroot 
I was unable to reopen #457 so I created new pull request.

File: src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs [hunk 1]
```diff
@@ -219,7 +429,15 @@ internal ResponseContext(string privilege)
...
-                _ResponseFetched?.Invoke(null, new RequestResponseEventArgs() { cause = _cause, result = _result, privilege = _privilege });
+                _ResponseFetched?.Invoke(null, new RequestResponseEventArgs { cause = _cause, result = _result, privilege = _privilege });
+            }
+
+            /// <summary>
+            /// ResponseContext class destructor
+            /// </summary>
+            ~ResponseContext()
+            {
+                s_responseMap.Remove(_privilege);
```
Comment (myroot) [inline src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs]: It is wrong
if contain this object in s_responseMap, it never called destructor. 
So, you does not need to implement this destructor

Comment (xerrni) [reply src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs]: Done, I have removed it.

File: src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs [hunk 2]
```diff
@@ -219,7 +429,7 @@ internal ResponseContext(string privilege)
 
             internal void FireEvent(CallCause _cause, RequestResult _result)
             {
-                _ResponseFetched?.Invoke(null, new RequestResponseEventArgs() { cause = _cause, result = _result, privilege = _privilege });
+                _ResponseFetched?.Invoke(null, new RequestResponseEventArgs { cause = _cause, result = _result, privilege = _privilege });
```
Comment (myroot) [inline src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs]: Why sender was null? it should be `this`(ResponseContext)

Comment (xerrni) [reply src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs]: This  bug was introduced in first Tizen.Security.PrivacyPrivilegeManager API implementation.
I fixed it now.

**Comment by xerrni** [comment]
@myroot
I have fixed issue with responseFetched.
Can you please merge this PR?

**Comment by myroot** [comment]
@xerrni I now merge, actually merge request should be done by F/W part, next time please ask to HQ

**Pull request merged**